RELEASE-NOTES.txt

```diff
@@ -12,6 +12,8 @@
 - [*] Stats: Fixed a crash when order stats use a date and time matching the start of Daylight Saving Time. [https://github.com/woocommerce/woocommerce-ios/pull/9083]
 - [*] Fix: Dismiss Take Payment popup after sharing the payment link to another app. [https://github.com/woocommerce/woocommerce-ios/pull/9042]
 - [*] Site credential login: Catch invalid cookie nonce [https://github.com/woocommerce/woocommerce-ios/pull/9102]
+- [*] Better error messages for site credential login failures [https://github.com/woocommerce/woocommerce-ios/pull/9125]
+- [Internal] New Zendesk tag for site credential login errors [https://github.com/woocommerce/woocommerce-ios/pull/9150]
```

Comment:
@spencertransier @itsmeichigo given the merge conflict resolution commits that removed the Zendesk tag, should this entry be removed as well?

Reply:
The feature was removed from this branch, but it was merged and built for 12.7.0.1 before, so the release note should still be valid for 12.7 right?